.github/workflows/build.yml

```diff
@@ -16,16 +16,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4', '8.0']
+        php-versions: ['7.4', '8.0', '8.1']
         dependencies: ['highest']
         composer-arguments: [''] # to run --ignore-platform-reqs in experimental builds
         static-analysis: ['no']
         experimental: [false]
-        include:
-          - php-versions: '7.3'
-            static-analysis: 'psalm'
-            experimental: true
-            dependencies: 'highest'
+        #include:
+          #- php-versions: '7.3'
+          #  static-analysis: 'psalm'
+          #  experimental: true
+          #  dependencies: 'highest'
```

Comment:
I'd say: remove those lines.. If we want them back, we can always look them up in the git history

Reply:
Since iirc the psalm checks failed before anyway, I'd like to leave them out for now, as I'd like this merged independently of us getting psalm back to life. Also, I never worked with psalm before, so I won't be able to help here

Reply:
Okay, we'll deal with that in #2704